draft-ietf-tls-iana-registry-updates.md

```diff
@@ -217,52 +263,122 @@ TLS_ECDHE_PSK_WITH_CHACHA20_POLY1305_SHA256   | {0xCC,0xAC}
+[SHALL update/has updated] the TLS Cipher Suite registry's policy as
+follows: 
 
-- Change the registry policy to:
+    Values in the range 0-223 are assigned via Specification Required
+    {{RFC8126}}.  Values 224-255 are reserved for Private Use.
 
-    Values in the range 0-223 are assigned via Specification Required {{RFC5226}}.  Values 224-255 are reserved for Private Use.
+See {{expert-pool}} for additional information about the designated
+expert pool.
 
-- Add the following:
+IANA [SHALL add/has added] the following notes:
 
-    Note: The designated expert {{RFC5226}} only ensures that the specification is publicly available.
+    Note(1): The designated expert {{RFC8126}} only ensures that the
+    specification is publicly available.
+
+    Note(2): ClientCertificateType Identifers marked as "Yes" are
+    those allocated via Standards Track RFCs.  ClientCertificateTypes
+    marked as "No" are not.
 
 New Session Ticket TLS Handshake Message Type
 =============================================
 
-To align with TLS implementations and to align the naming nomenclature for other Handshake message types, IANA is to rename entry 4 in the TLS HandshakeType registry to "new_session_ticket (renamed from NewSessionTicket)". IANA is to also add a reference to this document in the Reference column for entry 4 in the TLS HandshakeType registry.
+To align with TLS implementations and to align the naming nomenclature
+with other Handshake message types, IANA:
+
+- [SHALL rename/has renamed] entry 4 in the TLS HandshakeType registry
+to "new_session_ticket (renamed from NewSessionTicket)" {{!RFC5077}}.
+
+- [SHALL add/has added] a reference to this document in the Reference
+column for entry 4 in the TLS HandshakeType registry.
 
 TLS Exporter Label Registry
 ===========================
 
-IANA is to add the following note to the TLS Exporter Label Registry:
-
-    Note: {{RFC5705}} defines keying material exporters for TLS in terms of the TLS PRF. {{I-D.ietf-tls-tls13}} replaced the PRF with HKDF, thus requiring a new construction. The exporter interface remains the same, however the value is computed different.
+To aid those reviewers who start with the IANA registry, IANA [SHALL
+add/has added]:
 
-IANA is to also to add a "Recommended" column to the TLS Exporter Label registry.  The table that follows has been generated by marking Standards Track RFCs as "Yes" and all others as "No".  Future exporters MUST define the value of this column.  A Standards Track document {{RFC5226}} is required to register an extension with the value "Yes".
+- The following note to the TLS Exporter Label Registry:
 
-IANA is also to add the following note:
+    Note(2): {{!RFC5705}} defines keying material exporters for TLS in
+    terms of the TLS PRF.  {{I-D.ietf-tls-tls13}} replaced the PRF with
+    HKDF, thus requiring a new construction. The exporter interface
+    remains the same, however the value is computed different.
```

Comment:
This isn't correct. TLS 1.3 added a small difference. An empty context is no longer distinguishable from an absent one.

Reply:
Hmm... I copied this from s7.5 of the TLS1.3 draft.  So let's make sure we make the same changes.

Reply:
Hmm, 7.5 reads:

> If no context is provided, the context_value is zero-length. Consequently, providing no context computes the same value as providing an empty context. This is a change from previous versions of TLS where an empty context produced a different output to an absent context. As of this document’s publication, no allocated exporter label is used both with and without a context. Future specifications MUST NOT define a use of exporters that permit both an empty context and no context with the same label. 

Which is probably relevant.  Did you copy from an older version?

Reply:
I just copied the 1st para.